definitions/ext-db2/definition.yml

```diff
@@ -0,0 +1,18 @@
+domain: EXT
+type: DB2
+synthesis:
+  name: database
```

Comment:
@srvaroa what do you think about this?
I noticed other DB's (redis for example) use instances not databases for entities. Not all entries of DB2 or Oracle bring a instance though.

Is ok that we use instances on some DB's and database in others? not sure if there's a standard we are skipping here or not.